### ISSUE TITLE
Minified or compact version of JSON possible?

### ISSUE THREAD

**Issue by JannemanDev** [ttu/json-flatfile-datastore#71]
Currently when JSON objects are stored in the file it looks like this:
```json
{
  "priceData": [
    {
      "dateTime": "2022-10-06T12:05:00.834994+02:00",
      "exchangeName": "NYSE",
      "coinPair": "BLA/USD",
      "price": 1.1,
      "bid": 1.0,
      "ask": 1.2
    },
    {
      "dateTime": "2022-10-06T12:05:00.834994+02:00",
      "exchangeName": "NYSE",
      "coinPair": "BLA/USD",
      "price": 1.1,
      "bid": 1.0,
      "ask": 1.2
    }
  ]
}
```

This way the file gets very bloated. Is there a way to store it minified or compact like this:
```json
{"priceData":[{"dateTime":"2022-10-06T12:02:34.4853061+02:00","exchangeName":"NYSE","coinPair":"BLA/USD","price":1.1,"bid":1.0,"ask":1.2},{"dateTime":"2022-10-06T12:05:00.834994+02:00","exchangeName":"NYSE","coinPair":"BLA/USD","price":1.1,"bid":1.0,"ask":1.2}]}
```

**Comment by nabeelio** [comment]
Thanks @BlackBooth and @ttu! I was thinking of picking this up.

**Comment by ttu** [comment]
@nabeelio thanks for your interest! This was already implemented in #73 

I just forgot to close this issue 😞
